Parsers/ASimProcess/ProductParsers/ProcessEventMicrosoftLinuxSysmonCreate.yaml

```diff
@@ -0,0 +1,107 @@
+      '</Data><Data Name="TerminalSessionId">'TerminalSessionId
+      '</Data><Data Name="IntegrityLevel">'IntegrityLevel
+      '</Data><Data Name="Hashes">'Hashes
+      '</Data><Data Name="ParentProcessGuid">{'ParentProcessGuid
+      '}</Data><Data Name="ParentProcessId">'ParentProcessId
+      '</Data><Data Name="ParentImage">'ParentImage
+      '</Data><Data Name="ParentCommandLine">'ParentCommandLine
+      '</Data>' *
+  | extend    TargetProcessSHA1=extract(@'SHA1=(\w+)',1, tostring(Hashes)),
+              TargetProcessSHA256=extract(@'SHA256=(\w+)',1, tostring(Hashes)),
+              TargetProcessIMPHASH=extract(@'IMPHASH=(\w+)',1,tostring(Hashes)), // add to the empty schema + Excel file
+              TargetProcessMD5=extract(@'MD5=(\w+)',1, tostring(Hashes))
+      // End of XML parse
+  | extend 
+              EventType = "ProcessCreated",
+              EventStartTime = todatetime(TimeGenerated),
+              EventEndTime = todatetime(TimeGenerated),
+              EventCount = int(1),
+              EventVendor = "Microsoft",
+              EventSchemaVersion = "0.1.0",
+              EventProduct = "Event",
+              DvcOs = "Linux", // Changed
+              TargetUserSessionId = toint(LogonId)  ,           
+              TargetUsernameType = "Simple",
+              // ActorUsernameType = "Simple",
+              TargetUsername = User,
+              TargetProcessCommandLine = CommandLine,
+              EventOriginalId = '1' // Set with a constant value to avoid parsing
+
+  | project-rename 
+              // EventMessage = RenderedDescription, // field not available in Linux
+              DvcHostName = Computer, // Computer may be different than HostName, in which case HostIP may be incorrect. 
+              DvcIpAddr = HostIP, 
+              EventOriginalUid = EventRecordId,
+
+              TargetUserSessionGuid = LogonGuid, 
+
+              TargetProcessId = ProcessId,
+              TargetProcessGuid = ProcessGuid,
+              TargetProcessName = Image,
+              TargetProcessIntegrityLevel = IntegrityLevel,
+              
+              TargetProcessCompany = Company,
+              TargetProcessFileDescription = Description,
+              TargetProcessFileVersion = FileVersion,
+              TargetProcessFileProduct = Product,
+              
+              ActingProcessId = ParentProcessId,
+              ActingProcessGuid = ParentProcessGuid,                          
+              ActingProcessCommandLine = ParentCommandLine,
+              ActingProcessName = ParentImage
+
+              // ActorUsername = UserName // Not provided in Linux
+
+  | extend // aliases
+              // User = TargetUserName,
+              Process = TargetProcessName,
+              Dvc = DvcHostName,
+              Hash = coalesce(TargetProcessSHA256, TargetProcessSHA1, TargetProcessMD5) // which appears first - will be aliases to "Hash"
+
+  | project-away
+              ProcessName, ProcessID
```

Comment:
I do opt to remove confusing original fields. ProcessName is not the expected ProcessName: it is the name of the Sysmon process itself. ProcessId might be OK, but I felt that since we did not define if ProcessId is the Target or Acting one, I should remove it.